<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelSelectionPage_SMP</class>
 <widget class="QWizardPage" name="ModelSelectionPage_SMP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="title">
   <string>Select Model Architecture</string>
  </property>
  <property name="subTitle">
   <string>Choose the decoder architecture and encoder backbone for Segmentation Models PyTorch.</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QGroupBox" name="decoderGroupBox">
     <property name="title">
      <string>Decoder (Architecture)</string>
     </property>
     <layout class="QVBoxLayout" name="decoderLayout">
      <item>
       <widget class="QComboBox" name="decoderCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="encoderGroupBox">
     <property name="title">
      <string>Encoder (Backbone)</string>
     </property>
     <layout class="QVBoxLayout" name="encoderLayout">
      <item>
       <widget class="QComboBox" name="encoderCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pretrainedEncoderCheckbox">
     <property name="text">
      <string>Use ImageNet pretrained encoder weights (Recommended - faster training)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="estimatesLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { background-color: #f0f0f0; padding: 10px; border-radius: 5px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
